### ISSUE TITLE
Missing MySQL help command table causes background completion error

### ISSUE THREAD

**Issue by WinstonN** [dbcli/mycli#485]
Hi Hi

Using v 1.12.0 on Ubuntu 14.04 
on connecting to mysql I get error

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mycli/completion_refresher.py", line 61, in _bg_refresh
    refresher(completer, executor)
  File "/usr/local/lib/python2.7/dist-packages/mycli/completion_refresher.py", line 121, in refresh_show_commands
    completer.extend_show_items(executor.show_candidates())
  File "/usr/local/lib/python2.7/dist-packages/mycli/sqlcompleter.py", line 98, in extend_show_items
    for show_item in show_items:
  File "/usr/local/lib/python2.7/dist-packages/mycli/sqlexecute.py", line 197, in show_candidates
    cur.execute(self.show_candidates_query)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/cursors.py", line 166, in execute
    result = self._query(query)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 856, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 1057, in _read_query_result
    result.read()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 1340, in read
    first_packet = self.connection._read_packet()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 1014, in _read_packet
    packet.check_error()
  File "/usr/local/lib/python2.7/dist-packages/pymysql/connections.py", line 393, in check_error
    err.raise_mysql_exception(self._data)
  File "/usr/local/lib/python2.7/dist-packages/pymysql/err.py", line 107, in raise_mysql_exception
    raise errorclass(errno, errval)
ProgrammingError: (1146, u"Table 'mysql.help_topic' doesn't exist")
```

**Comment by tsroten** [comment]
@WinstonN Hi! I'm sorry you're having trouble using mycli.

The error is due to a MySQL system table not existing (`help_topic`). Do you know what version the MySQL server you are connecting to is?

If you can connect with the official `mysql` command-client, you can run the command `status` and it will tell you the server version. Here is an example server version: `5.6.33-0ubuntu0.14.04.1-log (Ubuntu)`

**Comment by WinstonN** [comment]
@tsroten no apologies needed my friend, you are great for writing such a beautiful tool! 

I'm using `mysql  Ver 14.14 Distrib 5.7.18, for Linux (x86_64) using  EditLine wrapper`

**Comment by tsroten** [comment]
It looks like the `help_topic` database isn't always installed. Thanks for reporting this. We'll get this fixed 😄 .

**Comment by tsroten** [comment]
@WinstonN This is fixed in mycli 1.12.1, which is available via pip:
```bash
pip install -U mycli
```

Thanks for reporting this!

**Comment by WinstonN** [comment]
Works! Thank you!
